packages/gestalt/src/IconButton.tsx

```diff
@@ -212,12 +256,11 @@ const IconButtonWithForwardRef = forwardRef<HTMLButtonElement, Props>(function I
       type={type === 'submit' ? 'submit' : 'button'}
     >
       <div
-        className={classnames(styles.button, touchableStyles.tapTransition, {
-          [styles.disabled]: disabled,
-          [styles.enabled]: !disabled,
-          [touchableStyles.tapCompress]: !disabled && isTapping,
-        })}
-        style={compressStyle || undefined}
+        className={isInVRExperiment ? vrStyles : divStyles}
+        style={{
+          ...(!isInVRExperiment && compressStyle),
```

Comment:
why 261?

Reply:
I am removing these tap feedback animations unless stated by the designs since they change the size based on classic sizes. Should they be left in in this case?

Reply:
Asked Kate, she confirmed these should be left in for now, while motion stuff is WIP